exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -554,12 +554,16 @@ func (c *CoinbasePro) SubmitOrder(ctx context.Context, s *order.Submit) (*order.
 			fpair.String(),
 			"")
 	case order.Limit:
+		timeInForce := ""
+		if s.ImmediateOrCancel {
+			timeInForce = string(CoinbaseRequestParamsTimeIOC)
+		}
 		orderID, err = c.PlaceLimitOrder(ctx,
 			"",
 			s.Price,
 			s.Amount,
 			s.Side.Lower(),
-			"",
+			timeInForce,
```

Comment:
Change type from string to `RequestParamsTimeForceType` to be explicit or alternatively you can to reduce type conversions:

```go
const CoinbaseRequestParamsTimeIOC = "IOC"
```

Reply:
done